### PR TITLE
[SES-225] Correct size dot and add chart stories

### PR DESCRIPTION
**Pull request by alejandrocabriales**
# Ticket
https://trello.com/c/fnTy5yik/225-feature-coreunittransparencyreporting-v5

# Actions
Correct size dot and some style

### CODE REVIEW

File: src/stories/containers/finances-overview/Components/ExpensesChart/ExpensesChart.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+      {
+        value: new TotalExpenseReportsBuilder().withDiscontinued(801576).build().discontinued,
+        // value: new TotalExpenseReportsBuilder().withDiscontinued(596316).build().discontinued,
+        itemStyle: {
+          borderRadius: [6, 6, 0, 0],
+        },
+      },
+      {
+        value: new TotalExpenseReportsBuilder().withDiscontinued(855260).build().discontinued,
+        // value: new TotalExpenseReportsBuilder().withDiscontinued(650000).build().discontinued,
```
Comment (YasielCabrera) [inline src/stories/containers/finances-overview/Components/ExpensesChart/ExpensesChart.stories.tsx:125]: are these comments required?

File: src/stories/containers/finances-overview/Components/ExpensesChart/ExpensesChart.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+        },
+      },
+      {
+        value: new TotalExpenseReportsBuilder().withDiscontinued(172334).build().discontinued,
+        itemStyle: {
+          // borderRadius: [0, 0, 6, 6],
+        },
+      },
+      {
+        // value: new TotalExpenseReportsBuilder().withDiscontinued(0).build().discontinued,
```
Comment (YasielCabrera) [inline src/stories/containers/finances-overview/Components/ExpensesChart/ExpensesChart.stories.tsx:162]: Are these comments required?

File: src/stories/containers/finances-overview/Components/ExpensesChart/ExpensesChart.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,291 @@
...
+        options: {
+          componentStyle: {
+            width: 607,
+            height: 387,
+          },
+          style: {
+            top: -2,
+            left: 3,
+            width: 607,
+            height: 382,
```
Comment (YasielCabrera) [inline src/stories/containers/finances-overview/Components/ExpensesChart/ExpensesChart.stories.tsx:285]: The `options > style` styles are for the Figma image container, it is now a good idea to set `width` and `height` here as the image should have the intrinsic size

File: src/stories/containers/finances-overview/Components/ExpensesChart/ExpensesChart.tsx [hunk 4]
```diff
@@ -47,14 +49,15 @@ const ExpensesChart: React.FC<Props> = ({ newActual, newDiscontinued, newPredict
...
+        fontSize: upTable ? 12 : 9,
+        height: upTable ? 15 : 11,
         baseline: 'top',
         interval: 0,
       },
     },
     yAxis: {
       axisLabel: {
+        margin: upTable ? 16 : 7,
         // eslint-disable-next-line spellcheck/spell-checker
```
Comment (YasielCabrera) [inline src/stories/containers/finances-overview/Components/ExpensesChart/ExpensesChart.tsx:61]: This this `eslint-disable` required? Maybe if `formatted` is being detected as a typo it should be added to the dictionary

**Review by YasielCabrera — CHANGES_REQUESTED** [review]

File: src/stories/containers/finances-overview/Components/ExpensesChart/ExpensesChart.stories.tsx (hunk 2 above)
Comment (alejandrocabriales) [reply src/stories/containers/finances-overview/Components/ExpensesChart/ExpensesChart.stories.tsx:162]: not my mistake

File: src/stories/containers/finances-overview/Components/ExpensesChart/ExpensesChart.stories.tsx (hunk 1 above)
Comment (alejandrocabriales) [reply src/stories/containers/finances-overview/Components/ExpensesChart/ExpensesChart.stories.tsx:125]: not may mistake

**Pull request merged**